### PR TITLE
Fixed the responsive home page

### PR DESCRIPTION
**Pull request by MostafaMohammadi1989**
Fixed reponsive each section on the home page

### CODE REVIEW

File: src/app/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import Head from "next/head";
+
+export default function Home() {
+  return (
+    <div>
+      <Head>
+        <title>This page has a title 🤔</title>
```
Comment (Farhad33) [inline src/app/index.js]: what is this?

File: src/app/layout.js [hunk 2]
```diff
@@ -3,19 +3,26 @@ import { Quicksand } from 'next/font/google'
...
 
 
 const  quicksand = Quicksand({
   subsets: ['latin'],
   weight: ['300', '400', '500', '600', '700']
 })
 
-
-
+// export const metadata = {
```
Comment (Farhad33) [inline src/app/layout.js]: remove the comments

File: src/app/index.js (hunk 1 above)
Comment (MostafaMohammadi1989) [reply src/app/index.js]: I forget eraser that

File: src/app/layout.js [hunk 3]
```diff
@@ -3,19 +3,26 @@ import { Quicksand } from 'next/font/google'
...
+//   title: 'Vfix Voip',
+//   description: 'Voip',
+// }
 
 export default function RootLayout({ children }) {
   return (
     <html lang="en">
+      <Head>
+      {/* <link rel="icon" href="app/favicon.ico" sizes="any" /> */}
```
Comment (Farhad33) [inline src/app/layout.js]: remove the comments

File: src/common/Theme.jsx [hunk 4]
```diff
@@ -61,30 +65,21 @@ export const Theme ={
...
-            subline1: "24px",
-            subline2: "20px",
-            subline3: "calc(1vw + 5px)",
-            body1: "24px",
-            body2: "calc(1vw + 5px)",
-            body3: "calc(1vw + 1px)",
-            button: "calc(1vw + 5px)",
-            caption: "20px",
-            overLine: "calc(1vw + -1px)",
+        //     h1: "700",
```
Comment (Farhad33) [inline src/common/Theme.jsx:68]: what happened here?

**Review by Farhad33 — APPROVED** [review]

**Pull request merged**